scripts/files/tests/fs_s3_test.py

```diff
@@ -124,3 +124,20 @@ def test_exists_object_starting_with_not_exists() -> None:
     file_exists = exists("s3://testbucket/hello/another.fi")
 
     assert file_exists is False
+
+
+@mock_s3  # type: ignore
+def test_list_files_in_uri() -> None:
+    bucket_name = "testbucket"
+    s3 = resource("s3", region_name=DEFAULT_REGION_NAME)
+    boto3_client = client("s3", region_name=DEFAULT_REGION_NAME)
+    s3.create_bucket(Bucket=bucket_name)
+    boto3_client.put_object(Bucket=bucket_name, Key="data/collection.json", Body=b"")
+    boto3_client.put_object(Bucket=bucket_name, Key="data/image.tiff", Body=b"")
+    boto3_client.put_object(Bucket=bucket_name, Key="data/image_meta.xml", Body=b"")
+
+    files = list_files_in_uri("s3://testbucket/data/", [".json", "_meta.xml"], boto3_client)
```

Comment:
We should probably use `bucket_name` here:

```suggestion
    files = list_files_in_uri(f"s3://{bucket_name}/data/", [".json", "_meta.xml"], boto3_client)
```